sdk/scala-sdk/src/main/scala/kalix/scalasdk/action/Action.scala

```diff
@@ -146,6 +145,14 @@ object Action {
        */
       def asyncEffect[S](futureEffect: Future[Action.Effect[S]]): Action.Effect[S]
 
+      /**
+       * Ignore the current element and proceed with processing the next element if returned for an element from
+       * eventing in. If used as a response to a regular gRPC or HTTP request it is turned into a NotFound response.
+       *
+       * @return
+       * @param S
+       */
+      def ignore[S](): Action.Effect[S]
```

Comment:
It's not imperative ignore, more of a value, so no parenthesis's?

Reply:
I wasn't sure. Yes, it makes sense.  I'll follow these two rules
- it has no side effects. See https://docs.scala-lang.org/style/method-invocation.html, see Arity-0 
- Also to hide the fact that it can be a method or a value. Following the uniform access syntax. See https://docs.scala-lang.org/glossary/index.html#:~:text=The%20uniform%20access%20principle%20states,call%20sites%20of%20parameterless%20functions.